reduction/imaging_parameters.py

```diff
@@ -2696,28 +2696,203 @@
+		"negativethreshold": 15.0,
+		"minbeamfrac": 0.3,
+		"dogrowprune": True,
+		"threshold": "2mJy"
+	},
     "G337.92_B6_12M_robust0": {
         "threshold": "14mJy",
         "startmodel": "G337.92_B6_uid___A001_X1296_X13b_continuum_merged_12M_robust0_selfcal4_finaliter",
     },
-    "G333.60_B3_12M_robust0": {"niter": 500000, "scales": [0, 3, 9, 27]},
-    "G333.60_B3_12M_robust0_contsub": {"niter": 500000, "scales": [0, 3, 9, 27]},
+	"G338.93_B3_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "G338.93_B3_uid___A001_X1296_X15b_continuum_merged_12M_robust0_selfcal2_finaliter"
+	},
+	"G338.93_B6_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "G338.93_B6__continuum_merged_12M_robust0_selfcal6_finaliter"
+	},
+	"G351.77_B3_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "G351.77_B3_uid___A001_X1296_X209_continuum_merged_12M_robust0_selfcal4_finaliter"
+	},
+	"G351.77_B6_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "G351.77_B6_uid___A001_X1296_X201_continuum_merged_12M_robust0_selfcal4_finaliter"
+	},
+	"G353.41_B3_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "G353.41_B3_uid___A001_X1296_X1d5_continuum_merged_12M_robust0_selfcal3_finaliter"
+	},
+	"G353.41_B6_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "G353.41_B6_uid___A001_X1296_X1cb_continuum_merged_12M_robust0_selfcal3_finaliter"
+	},
+	"W43-MM1_B3_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "W43-MM1_B3_uid___A001_X1296_X1ad_continuum_merged_12M_robust0_selfcal4_finaliter"
+	},
+	"W43-MM2_B3_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "W43-MM2_B3_uid___A001_X1296_X11f_continuum_merged_12M_robust0_selfcal4_finaliter"
+	},
+	"W43-MM2_B6_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "W43-MM2_B6_uid___A001_X1296_X111_continuum_merged_12M_robust0_selfcal5_finaliter"
+	},
+	"W43-MM3_B3_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "W43-MM3_B3_uid___A001_X1296_X12f_continuum_merged_12M_robust0_selfcal4_finaliter"
+	},
+	"W43-MM3_B6_12M_robust0": {
+		"threshold": "6mJy"
+		"startmodel": "W43-MM3_B3_uid___A001_X1296_X12f_continuum_merged_12M_robust0_selfcal4_finaliter"
+	},
     "W51-E_B6_12M_robust0": {
+        "usemask": "auto-multithresh",
+        "sidelobethreshold": 1.0,
         "pblimit": 0.1,
         "threshold": "16mJy",  # sigma is ~4 mJy
         "startmodel": "W51-E_B6_uid___A001_X1296_X213_continuum_merged_12M_robust0_selfcal6_finaliter",
     },
-    "W51-E_B6_12M_robust0_contsub": {"pblimit": 0.1, "threshold": "16mJy",},  # sigma is ~4 mJy
+    "W51-E_B6_12M_robust0_contsub": {
+        "usemask": "auto-multithresh",
+        "sidelobethreshold": 1.0,
+        "pblimit": 0.1,
+        "threshold": "16mJy",  # sigma is ~4 mJy
+    },
     "W51-E_B3_12M_robust0": {
-        "startmodel": "W51-E_B3_uid___A001_X1296_X10b_continuum_merged_12M_robust0_selfcal4",
+        "startmodel": "W51-E_B3_uid___A001_X1296_X10b_continuum_merged_7M_robust0_selfcal4_finaliter",
```

Comment:
```suggestion
        "startmodel": "W51-E_B3_uid___A001_X1296_X10b_continuum_merged_12M_robust0_selfcal4",
```
don't use 7m startmodel!

Reply:
I made this mistake earlier.  It's because the 12m was missing at some stage